include/vara/Feature/FeatureModel.h

```diff
@@ -34,8 +34,6 @@ class FeatureModel {
 
 public:
   using FeatureMapTy = llvm::StringMap<std::unique_ptr<Feature>>;
-  using ConstraintTy = Constraint;
-  using ConstraintContainerTy = std::vector<std::unique_ptr<ConstraintTy>>;
   using RelationshipContainerTy = std::vector<std::unique_ptr<Relationship>>;
```

Comment:
Ok, maybe I don't get the design, but why is constraint gone? But a class added below?